jwt/jwt.go

```diff
@@ -154,3 +192,66 @@ func Decode(_ context.Context, params *DecodeParams) (*clerk.Claims, error) {
 		Extra:  extraClaims,
 	}, nil
 }
+
+// Caching store.
+type cache struct {
+	mu      sync.RWMutex
+	entries map[string]*cacheEntry
+}
+
+// Get returns the cache entry for the provided key, if one exists.
+func (c *cache) Get(key string) (*cacheEntry, bool) {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	entry, ok := c.entries[key]
```

Comment:
❓ If the entry is expired, I think you can ignore it here and don't return it?

Reply:
Was debating this as well. The entry is still there technically, but I guess users of a cache would expect to not get back expired entries?

Reply:
If it's going to be replaced, sounds good to me to not return it. My suggestion just transfers the concern to the cache layer. Not a big deal.